packages/shopify-codemod/transforms/remove-empty-catch-blocks.js

```diff
@@ -0,0 +1,16 @@
+export default function removeEmptyCatchBlocks({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.TryStatement, {
+      type: 'TryStatement',
+      handler: {
+        type: 'CatchClause',
+      },
+    })
+      .forEach(({node}) => {
+        if (node.handler.body.body.length === 0) {
+          delete node.handler;
```

Comment:
Also, let's try this using the `NodePath` methods — `path.get('handler').replace()`